ci/build.yml

```diff
@@ -0,0 +1,21 @@
+---
+
+platform: linux
+
+image_resource:
+  type: docker-image
+  source:
+    repository: golang
+    tag: 1.16.3
```

Comment:
A new patch version 1.16.4 has been released which includes security fix and to net/http packages. Will it work to our advantage?